doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
+A backend allows a file to be interpreted as keys in the key database such that any edits to the keys are reflected in the file and vice versa.  
+Additionally, the user can use this command to list the currently mounted backends by running the command with no arguments.  
+
+## USAGE
+
+`kdb mount [<path> <mountpoint>] [<plugin> [<config>] [..]]`  
```

Comment:
The mount documentation doesn't seem like a good place for a full explanation of this. I think the documentation can be merged now. This could be something covered in the `doc` directory at a higher level.